src/server/api/routes/users.router.js

```diff
@@ -0,0 +1,60 @@
+const express = require('express');
```

Comment:
This was Charles task. This is exactly why communication is **so** important. 

Now Charles will have wasted his time implementing a task you did for him. Have you talked together about this?

Because if we use this, then i will have to delete his task and tell him to work on something else

Reply:
I think Charles task don't finished, I only start. Because I return in database only uid, email and username, another columns should be added.

Reply:
Okay, but make sure you talk with him about this, so he is not developing the same things